Source/WebCore/platform/graphics/texmap/coordinated/CoordinatedBackingStoreTile.cpp

```diff
@@ -48,13 +48,13 @@ void CoordinatedBackingStoreTile::processPendingUpdates(TextureMapper& textureMa
     if (!updatesCount)
         return;
 
-    WTFBeginSignpost(this, CoordinatedSwapBuffers, "%lu updates", updatesCount);
+    WTFBeginSignpost(this, CoordinatedSwapBuffers, "%zu updates", updatesCount);
```

Comment:
Would it make more sense to use the PRui64 (not sure this is the right one, always getting confused with the letters) formatters?

Reply:
Here `"%zu"` is the correct format because the variable has type `size_t`, which might be 32-bit in some architectures. Using `"%"PRIu64` should be done only when the variable is definitely known to be 64-bit wide like `uint64_t`.

Reply:
PRInt64 makes code super hard to read. And even it is not stable for various other type usage like size_t. I would like to just have "%llu" and cast uint64_t in the parameter.

Reply:
Unfortunately it still has to be llu on 32-bit, and lu on 64-bit. So we have to choose between a cast + PRIu64, or zu

Reply:
Note that `%zu` will automatically match the width-size of a `size_t` for the current platform, no need for `PRIxxx` macros, nor to choose between `%lu` and `%llu`. Rule of thumb: use always `%zu` for `size_t` and `%zd` for `ssize_t`.